modules/consensus/consensusset.go

```diff
@@ -100,12 +100,12 @@ func New(gateway modules.Gateway, persistDir string) (*ConsensusSet, error) {
 
 		dosBlocks: make(map[types.BlockID]struct{}),
 
-		clock: types.StdClock{},
-
 		marshaler: encoding.StdGenericMarshaler{},
 
 		blockRuleHelper: stdBlockRuleHelper{},
 
+		blockValidator: NewBlockValidator(),
```

Comment:
also, 'RuleHelper' feels like a weak name, but I don't have any good alternate suggestions at the moment.